sdk/storage/Azure.Storage.Blobs/AzureStorageNetMigrationV12.md

```diff
@@ -0,0 +1,264 @@
+# Migration Guide: From Microsoft.Azure.Storage.Blob to Azure.Storage.Blobs
+
+This guide intends to assist customers in migrating from the legacy version 11 of the Azure Storage .NET library for Blobs to version 12.
```

Comment:
Do we want to say v11 specifically or just "older versions?" Because we've dealt with people trying to upgrade from way older versions to v11 first and that has proven far more effort than it is worth just getting people up to another legacy version.

Reply:
I guess all our sidebyside examples will be v11, so maybe this isn't the correct move. Looking for other opinions on this.

Reply:
I might not be well-versed in the libaries older than v11 (and maybe even v11 too) but how different is say v9 from v11? If they're too different it might be best to save doing a migration guide for v9 to v12 for later. If not then we can try to look to adding a v9 example to this migration guide later.

Reply:
They aren't different enough to warrant a guide. But there are significant changes to internals, and anyone operating at high scale will have trouble adjusting over some major revs.

Reply:
There are customers using various versions (7,8,9,10,11,12) of the library but this migration guide contains code snippets in v11, that's why I specified it. I agree that if we want to add other code snippets we could generalize this statement to say "older versions". For now, we could maybe add to it saying v9 is similar and customers using v9 may find this guide helpful too.

Reply:
Tracking separately.